pom.xml

```diff
@@ -109,6 +109,12 @@
     <version.org.jboss.marshalling.api>1.2.3.GA</version.org.jboss.marshalling.api>
     <!-- Version 2.0.1.Final which is coming from kie-parent is not compatible with dashbuilder-validator module -->
     <version.javax.validation>1.0.0.GA</version.javax.validation>
+
+    <!-- Mock server testing later in the stack -->
+    <!-- Add this to match other versions of okhttp3 and okio used elsewhere -->
+    <version.com.squareup.okhttp3>4.12.0</version.com.squareup.okhttp3>
+    <version.com.squareup.okio>3.6.0</version.com.squareup.okio>
+    <version.org.bouncycastle>1.78.1</version.org.bouncycastle>
```

Comment:
@LightGuard Why we need to declare those versions here? Can't we use the same declared in the parent pom in [droolsjbpm-build-bootstrap](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2556/files) ?

Reply:
No idea. I have a duplicate class issue locally, which does not appear to be related. Are other builds green without this change?

Reply:
@LightGuard Related PRs get merged, thus I guess this PR is no longer necessary 
//cc @baldimir